index.html

```diff
@@ -0,0 +1,204 @@
+			</ul>
+		</div>
+	</section>
+
+	<!-- project -->
+	<section class="card">
+		<section class="page-2">
+			<div class="wrapper">
+				<div class="img">
+					<img src="./images/Page 2 image.png" alt="page 2 image">
+				</div>
+				<div class="text">
+					<h1>Tonic</h1>
+					<ul>
+						<h5>CANOPY</h5>
+						<li>Back End Dev</li>
+						<li>2015</li>
+					</ul>
+					<p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.
+					</p>
+				</div>
+				<ul class="btn">
+					<li><a href="#" class="html">HTML</a></li>
+					<li><a href="#" class="css">CSS</a></li>
+					<li><a href="#" class="j555avascript">JavaScript</a></li>
+				</ul>
+				<div class="container">
+					<a href="#">See project</a>
+				</div>
+			</div>
+		</section>
+
+		<section class="page-3">
+			<div class="wrapper-3">
+				<div class="img-3">
+					<img src="./images/Page 3 image.jpg" alt="page 3 image">
+				</div>
+				<div class="text">
+					<h1>Multi-Post<br>Stories</h1>
+					<ul>
+						<h5>CANOPY</h5>
+						<li>Back End Dev</li>
+						<li>2015</li>
+					</ul>
+					<p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.
+					</p>
+				</div>
+				<ul class="btn">
+					<li><a href="#" class="html">HTML</a></li>
+					<li><a href="#" class="css">CSS</a></li>
+					<li><a href="#" class="javascript">JavaScript</a></li>
+				</ul>
+				<div class="container">
+					<a href="#">See project</a>
+				</div>
+			</div>
+		</section>
+
+		<section class="page-4">
+			<div class="wrapper-4">
+				<div class="img-4">
+					<img src="./images/Page 4 image.jpg" alt="page 4 image">
+				</div>
+				<div class="text">
+					<h1>Tonic</h1>
+					<ul>
+						<h5>CANOPY</h5>
+						<li>Back End Dev</li>
+						<li>2015</li>
+					</ul>
+					<p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.
+					</p>
+				</div>
+				<ul class="btn">
+					<li><a href="#" class="html">HTML</a></li>
+					<li><a href="#" class="css">CSS</a></li>
+					<li><a href="#" class="javascript">JavaScript</a></li>
+				</ul>
+				<div class="container">
+					<a href="#">See project</a>
+				</div>
+			</div>
+		</section>
+
+		<section class="page-5">
+			<div class="wrapper-5">
+				<div class="img-5">
+					<img src="./images/Page 5 image.jpg" alt="page 5 image">
+				</div>
+				<div class="text">
+					<h1>Multi-Post<br>Stories</h1>
+					<ul>
+						<h5>CANOPY</h5>
+						<li>Back End Dev</li>
+						<li>2015</li>
+					</ul>
+					<p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.
+					</p>
+				</div>
+				<ul class="btn">
+					<li><a href="#" class="html">HTML</a></li>
+					<li><a href="#" class="css">CSS</a></li>
+					<li><a href="#" class="javascript">JavaScript</a></li>
+				</ul>
+				<div class="container">
+					<a href="#">See project</a>
+				</div>

```

Comment:
- [ ] Your button needs to have a margin-bottom, I kindly suggest you fix it 👍🏿 
- [ ] Your Design 
![image](https://github.com/Fentahunn/finish_mobile_version/assets/68823331/3f9b38b1-0a5b-4d16-a88b-7572f6698266)

- [ ] Figma Design
![image](https://github.com/Fentahunn/finish_mobile_version/assets/68823331/3e3875fa-4992-4074-b223-d3830ff0266a)

- [ ] Great job on implementing the button! However, in order to align with the design requirements, please make the following updates:

- Apply the necessary CSS classes or styles to the button element to achieve the active, hover, and pressed button styles as specified in the design.

- [ ] Here is the required button style for each event

![image](https://github.com/Fentahunn/finish_mobile_version/assets/68823331/c8336ade-f036-41e3-b50c-538497d4f6e8)